modules/programs/radio-cli.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}:
+let
+  inherit (lib)
+    mkIf
+    mkEnableOption
+    mkPackageOption
+    mkOption
+    ;
+
+  cfg = config.programs.radio-cli;
+  jsonFormat = pkgs.formats.json { };
+in
+{
+  meta.maintainers = with lib.hm.maintainers; [ aguirre-matteo ];
+
+  options.programs.radio-cli = {
+    enable = mkEnableOption "radio-cli";
+    package = mkPackageOption pkgs "radio-cli" { nullable = true; };
+    settings = mkOption {
+      type = jsonFormat.type;
```

```suggestion
      inherit (jsonFormat) type;
```

Just a nit to prevent linting warnings, in the future.

---

Ok!